README.md

```diff
@@ -22,14 +22,14 @@ and publishes it. This would test that functionality
 
 ```python
 
-from pyrostest import RosTest, launch_node, mock_node, check_topic
+from pyrostest import RosTest, launch_node, mock_node
 from std_msgs.msg import Float64
 
 class TestAddOne(RosTest):
     @launch_node('pkg', 'add_one')
     def test_add_one(self):
-        with mock_pub('/input', Flaot64, queue_size=0) as p:
-            with check_topic('/output', Float64) as r:
+        with self.mock_pub('/input', Flaot64, queue_size=0) as p:
+            with self.check_topic('/output', Float64) as r:
```

Comment:
The nice thing about this is now mock_pub and check_topic have access to the rosmaster api in case we care about that later.